### DISCUSSION TITLE
what termux version 0.180 not found f-droid

### DISCUSSION THREAD

**Discussion by ExtinctDrift19** [termux/termux-app#2519] [Q&A] [answered]
refused to publish  a new version f-droid termux

**Comment by agnostic-apollo** [comment, marked as answer]
Read why at https://github.com/termux/termux-app#f-droid

I already checked yesterday and `termux-app` was built successfully but has not been deployed yet. Deployment hasn't happened for today, at least currently, may happen tomorrow.

https://monitor.f-droid.org/builds/build
https://monitor.f-droid.org/builds/deploy
